src/renderer/pages/project/components/chat.vue

```
@@ -1,7 +1,7 @@
 <template>
   <div class="space-y-4">
     <!-- Chat history -->
-    <div ref="chatHistoryRef" class="bg-white border rounded-lg p-4 h-80 overflow-y-auto space-y-4">
+    <div ref="chatHistoryRef" class="bg-white border rounded-lg p-4 h-80 overflow-y-auto space-y-4" v-show="messages.length > 0">
```

ESLint

[error] Replace `·ref="chatHistoryRef"·class="bg-white·border·rounded-lg·p-4·h-80·overflow-y-auto·space-y-4"·v-show="messages.length·>·0"` with `⏎······ref="chatHistoryRef"⏎······class="bg-white·border·rounded-lg·p-4·h-80·overflow-y-auto·space-y-4"⏎······v-show="messages.length·>·0"⏎····`

(prettier/prettier)